test/jdk/javax/swing/JTableHeader/TableRendererTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/* @test
+   @bug 6429812
+   @key headful
+   @requires (os.family == "windows")
+   @library /java/awt/regtesthelpers
+   @build PassFailJFrame
+   @summary Test to check if table is printed without NPE
+   @run main TableRendererTest
+*/
```

Comment:
Could you move the test tags below the imports?

Most new tests follow this style and I find it more convenient when viewing the test source in the IDE, the test tags aren't collapsed this way.

Please add asterisk at the beginning of each line. It should look like this:

```java
/*
 * @test
 * @bug
 */
```

To follow the common style of Java block comments and test tags in the majority of files.

Please do not use `/**` as the first comment: it's not a Javadoc comment to avoid warning about the unknown javadoc tags.

Reply:
Updated.